src/lib/Components/Consignments/Submission/__tests__/objectToGraphQL-tests.ts

```diff
@@ -0,0 +1,33 @@
+import { objectToGraphQLInput } from "../objectToGraphQL"
+
+describe("JS -> GQL", () => {
+  it("handles some string", () => {
+    const result = `{ a: "a", b: "b" }`
+    const input = { a: "a", b: "b" }
+    expect(objectToGraphQLInput(input)).toEqual(result)
+  })
+
+  it("handles some numbers", () => {
+    const result = `{ a: 6, b: "b" }`
+    const input = { a: 6, b: "b" }
+    expect(objectToGraphQLInput(input)).toEqual(result)
+  })
+
+  it("ignores null values", () => {
+    const result = `{ a: 6 }`
+    const input = { a: 6, b: null }
+    expect(objectToGraphQLInput(input)).toEqual(result)
+  })
+
+  it("handles enums correctly", () => {
+    const result = `{ a: 6, b: NO, c: "NO" }`
+    const input = { a: 6, b: "NO", c: "NO" }
+    expect(objectToGraphQLInput(input, ["b"])).toEqual(result)
+  })
+
+  it("handles bools correctly", () => {
+    const result = "{ a: true, b: false }"
+    const input = { a: true, b: false, c: undefined }
+    expect(objectToGraphQLInput(input, ["b"])).toEqual(result)
+  })
+})
```

Comment:
JS object to the highlighted bit:

![screen shot 2017-09-14 at 3 38 35 pm](https://user-images.githubusercontent.com/49038/30452168-d0527db6-9962-11e7-8d89-c4b087948cd2.png)